### PR TITLE
Newsletter-214: Translate to German

### PR DESCRIPTION
**Pull request by tschweg**
@Xekyo thanks in advance for the review.

### CODE REVIEW

**Review by Xekyo — APPROVED** [review]
ACK, LGTM

File: _posts/de/newsletters/2022-08-24-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+{% include linkers/issues.md v=2 issues="25504,2234,1503,911,759,6596,1004" %}
+[core lightning 0.12.0]: https://github.com/ElementsProject/lightning/releases/tag/v0.12.0
+[bdk 0.21.0]: https://github.com/bitcoindevkit/bdk/releases/tag/v0.21.0
+[lnd 0.15.1-beta.rc1]: https://github.com/lightningnetwork/lnd/releases/tag/v0.15.1-beta.rc1
+[news212 bolts911]: /en/newsletters/2022/08/10/#bolts-911
+[aliases]: /en/newsletters/2022/07/13/#lnd-5955
+[woltx sp]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-August/020883.html
+[riard jam]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2022-August/003673.html
+[rn jam]: https://jamming-dev.github.io/book/
+[news210 commando]: /en/newsletters/2022/07/27/#core-lightning-5370
```
Comment (bitschmidty) [inline _posts/de/newsletters/2022-08-24-newsletter.md:139]:
```suggestion
[news210 commando]: /de/newsletters/2022/07/27/#core-lightning-5370
```

File: _posts/de/newsletters/2022-08-24-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  ermöglicht das Erkennen von [splices][topic splicing], worin ein Kanal nicht
+  wirklich geschlossen wird, sondern stattdessen weitere Mittel in einer
+  Onchain-Transaktion hinzugefügt oder entfernt werden.
+
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="25504,2234,1503,911,759,6596,1004" %}
+[core lightning 0.12.0]: https://github.com/ElementsProject/lightning/releases/tag/v0.12.0
+[bdk 0.21.0]: https://github.com/bitcoindevkit/bdk/releases/tag/v0.21.0
+[lnd 0.15.1-beta.rc1]: https://github.com/lightningnetwork/lnd/releases/tag/v0.15.1-beta.rc1
+[news212 bolts911]: /en/newsletters/2022/08/10/#bolts-911
```
Comment (bitschmidty) [inline _posts/de/newsletters/2022-08-24-newsletter.md:134]:
```suggestion
[news212 bolts911]: /de/newsletters/2022/08/10/#bolts-911
```

Comment (bitschmidty) [reply _posts/de/newsletters/2022-08-24-newsletter.md:134]: See https://github.com/bitcoinops/bitcoinops.github.io/pull/851 which enables this link to work.

@tschweg take a look if you have a second

**Review by bitschmidty — COMMENTED** [review]
Couple /de/ links that could be used instead of the /en/ version

**Comment by Xekyo** [comment]
Good catch, @bitschmidty. I took the liberty of making a commit to add this change.

File: _posts/de/newsletters/2022-08-24-newsletter.md (hunk 2 above)
Comment (tschweg) [reply _posts/de/newsletters/2022-08-24-newsletter.md]: @bitschmidty : Sorry for the late reply, I was out for a while. Thanks for catching the bad links! :+1:

**Pull request merged**